lib/chess/pieces/piece.rb

```diff
@@ -0,0 +1,116 @@
+require_relative '../chess'
+require_relative '../step'
+require 'colorize'
+require 'colorized_string'
+require 'byebug'
```

Comment:
what's not needed for _production_ shouldn't be permanent. I think these `require "byebug"` lines shouldn't be commited